style.css

```diff
@@ -258,25 +261,162 @@ body.lock {
     width: 40px;
     background-color: var(--main-bg-color);  
     border: 2px solid var(--main-border-color);  ;
-    font-size: 20px;
     right: 25px;
 }
-.arrow_left {
+.arrow-left {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(135deg);
 }
-.arrow_right {
+.arrow-right {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(-45deg);
 }
 .content {
-    padding: 70px 0 0;
+    margin: 60px 0 0;
+}
+.carousel {
+    position: relative;
+    width: 100vw;
+    height: 140vw;
```

Comment:
Why are you assigning 140vw to height?

Reply:
This is for correct proportions of these pictures (images in carousel have proportion 1:1.4)